.github/workflows/python-package.yml

```diff
@@ -25,11 +33,16 @@ jobs:
     
     - name: Install dependencies
       run: |
+        pushd scanpy
+        patch -p1 < ../scanpy-scripts/scrublet.patch
+        popd
+
         sudo apt-get install libhdf5-dev
         pip install -U setuptools>=40.1 wheel 'cmake<3.20'
-        pip install .
+        pip install $(pwd)/scanpy-scripts
```

Comment:
I don't mind currently a lot for this specific PR, but I would have thought that we were installing scanpy through conda for tests, I could imagine certain errors that happen with a conda installation and not with pip and the other way around... since we depend mostly on the conda route, it would make more sense in the mid term to have the CI installing it through conda.